src/judging_round_display.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+
+class JudgingRoundDisplay extends React.Component {
+    state = {
+        data: {},
+    }
+
+    componentDidUpdate(prevProps) {
+	const judging_round_id = this.props.judging_round
+	if (judging_round_id &&
+	      prevProps.judging_round !== this.props.judging_round) {
+            fetch("http://localhost:8000/api/v1/judging_round/" +
```

Comment:
This literal seems out of place. Obviously in django we'd get it from a reverse lookup, which we can't do here, but it seems like we should do something better here. 
At the very least, we probably don't want to point to `localhost:8000` ;)

Reply:
Yeah, I want to parameterize, but haven't gotten to it yet.  I want to model it after what they are doing in directory, but I didn't want to hold up this ticket on that.

Reply:
In the directory we've extracted a `util` function to make the api call: https://github.com/masschallenge/directory/blob/development/src/utils/fetchAlgoliaToken.js

If you did something similar this could become:

```js
componentDidUpdate(prevProps) {
  const judging_round_id = this.props.judging_round
  if (judging_round_id && prevProps.judging_round !== this.props.judging_round) {
    fetchJudgingRound(judging_round_id).then(data => { this.setState({ data: data }) })
  }
```

Reply:
Addressed.